src/ansys/fluent/core/launcher/pim_launcher.py

```diff
@@ -1,4 +1,16 @@
-"""Provides a module for launching Fluent in pim mode."""
+"""Provides a module for launching Fluent in pim mode.
+
+Examples
+--------
+
+>>> from ansys.fluent.core.launcher.launcher import create_launcher
+
+>>> pim_meshing_launcher = create_launcher("pim", mode="meshing")
+>>> pim_meshing_session = pim_meshing_launcher()
+
+>>> pim_solver_launcher = create_launcher("pim")
```

Comment:
Do we still have the enum? I think we should support it through this interface.

Reply:
Yes, done. Thank you.